src/main/java/Duke.java

```diff
@@ -1,10 +1,251 @@
+     */
+    public static void printError() {
+        printTopLine();
+        System.out.println(INDENT + "You didn't input the type of task. Again. Or you're stupid. Whatever.");
+        printBottomLine();
+    }
+
+    /**
+     * Prints error message for when user leaves out important information in input.
+     */
+    public static void printMissingTextError() {
+        printTopLine();
+        System.out.println(INDENT + "And?? Retype and complete your sentence like a grown adult. Please.");
+        printBottomLine();
+    }
+    /**
+     * Adds inputs from user to list[] to keep track of user's tasks, deadlines, and events.
+     *
+     * @param taskName Name of task from user.
+     * @param taskType Type of task from user.
+     * @param taskDetails Time/date of event/deadline.
+     */
+    public static void addTask(String taskName, String taskType, String taskDetails) {
+        switch (taskType) {
+        case "todo":
+            list[taskCount] = new Todo(taskName);
+            break;
+        case "deadline":
+            list[taskCount] = new Deadline(taskName, taskDetails);
+            break;
+        case "event":
+            list[taskCount] = new Event(taskName, taskDetails);
+            break;
+        default:
+            return;
+        }
+
+        taskCount++;
+        String plural = (taskCount - taskCompleted) == 1 ? "" : "s";
+
+        printTopLine();
+        System.out.println(INDENT + " Fine. Added to your list:");
+        System.out.println(INDENT + "   " + list[taskCount - 1]);
+        System.out.println(INDENT + " You have " + (taskCount - taskCompleted)
+                + " pending task" + plural + ". tHaT's aWeSoMe!!!1!!1!!");
+        printBottomLine();
+    }
+
+    /**
+     * Marks tasks in list[] as done, if they exist or has not been completed.
+     *
+     * @param taskNumber Number n associated with the task, where n is the nth task in list.
+     */
+    public static void markTaskAsDone(int taskNumber) {
+        boolean isExists = taskNumber >= 0 && taskNumber < taskCount;
+
+        printTopLine();
+        if (!isExists) {
+            System.out.println("    │ Wha- Hey! Task does not exist!");
+        } else if (list[taskNumber].isDone) {
+            System.out.println("    │ Dude... you've done the task already.");
+        } else if (isExists){
+            taskCompleted++;
+            list[taskNumber].isDone = true;
+            System.out.println("    │ About time. I've mark that task as done:");
+            System.out.println("    │   [" + list[taskNumber].getStatusIcon() + "]" + list[taskNumber].description);
+        }
+        printBottomLine();
+    }
+
+    /**
+     * Checks whether there exist an input after user input the command word.
+     *
+     * @param input Expected input from user after the command word taskType is typed.
+     * @return true if input is empty, false if input is not empty.
+     */
+    public static boolean isEmpty(String input) {
```

Comment:
Perhaps this method may not be necessary could be done once within a parsing method?
```Java
boolean isEmpty = input.equals("");
```